debian/rules

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+%:
+	dh $@ --parallel --with autoreconf
+
```

Comment:
I think we may need to override dh_auto_configure in order to build with both dsc and ebb enabled. Something like this:

override_dh_auto_configure:
	dh_auto_configure -- --enable-dsc --enable-ebb

EBB may be unset by default.